packages/genesys-spark-components/src/components/stable/gux-table/gux-table.tsx

```diff
@@ -79,6 +79,12 @@ export class GuxTable {
   @Prop()
   emptyMessage: string;
 
+  /**
+   * Indicates if the tables first column is fixed and will stay in place even when scrolling horizontally.
+   */
+  @Prop()
+  fixedFirstColumn: boolean = false;
```

Comment:
Not sure about this. What happens when the inevitable `fixedSecondColumn` is needed?
Can we discuss this at standup?